src/org/usfirst/frc/team467/robot/Robot.java

```diff
@@ -67,7 +72,7 @@ public void robotInit() {
 		// Initialize math lookup table
 		LookUpTable.init();
 
-		vision = VisionProcessing.getInstance();
+//		vision = VisionProcessing.getInstance();
```

Comment:
You should add a comment describing why this is commented out, so we know what to fix later.